AdtSampleApp/SampleClientApp/Program.cs

```diff
@@ -3,77 +3,45 @@
 using Azure.Identity;
 using Microsoft.Extensions.Configuration;
 using System;
-using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 
 namespace SampleClientApp
 {
     public class Program
     {
-        // Properties to establish connection
-        // Please copy the file serviceConfig.json.TEMPLATE to serviceConfig.json 
-        // and set up these values in the config file
-        private static string clientId;
-        private static string tenantId;
-        private static string adtInstanceUrl;
-
         private static DigitalTwinsClient client;
 
         static async Task Main()
         {
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
This was causing problems when not using the standard Windows console application.

Reply:
Unfortunately, it makes the app look ugly. when running. As people are using it in practice as a little tool, I'd prefer to not remove this code.

Reply:
This practically makes the app fault on my machine (I use cmder.exe as my console). Aside from it breaking some non standard shell users and not addressing Mac users or Linux hosts, I'm pretty sure that resizing someone's console is an accessibility no-no. I'm wondering if the formatting could be optimized without having to resize the console?